src/Translator.php

```diff
@@ -298,6 +298,25 @@ private function mergeYaml(): void
                 $contentYaml = $this->arrayMergeRecursive($contentYaml, $parsedYaml);
                 $this->recursiveKeySort($contentYaml);
             }
+            // Remove any keys where the value is the same as the source english value
+            $enPath = preg_replace('#/[a-zA-Z_]+\.yml#', '/en.yml', $path);
+            if ($enPath !== $path) {
+                $enYaml = Yaml::parse(file_get_contents($enPath));
+                foreach (array_keys($contentYaml) as $key01) {
+                    foreach (array_keys($contentYaml[$key01]) as $key02) {
```

Comment:
What are `$key01` and `$key02`? Please use variable names that describe what is contained in those variables. Otherwise, even if you explain what they are now, any time someone has to look at what's happening her in the future they'd have to either debug to see what's happening or come back to this PR to see what was said.

Reply:
If the variables themselves don't matter, but this is just indicative of the structure of the yaml content, please provide an example of what is happening as a code comment.

Reply:
Updated